skimage/segmentation/_join.py

```diff
@@ -114,20 +116,29 @@ def relabel_sequential(label_field, offset=1):
     >>> relab
     array([5, 5, 6, 6, 7, 9, 8])
     """
+    offset = int(offset)
+    if offset <= 0:
+        raise ValueError("Offset must be strictly positive.")
+    if int(label_field.min()) < 0:
```

Comment:
The `int` seems unnecessary? 

```suggestion
    if np.min(label_field) < 0:
```

Reply:
It is unnecessary if `label_field` is an integer array. But the code also covers the case where the input is of a different data type (e.g. float). That's why I thought it doesn't hurt to wrap it in `int()` just to be sure.

Reply:
Hmm. I'm surprised to find that the function indeed works with float types, but I guess if it does, let's not break it. However, `int(-0.01)` is 0, and I wonder whether I'd rather we raised in that case...

Reply:
I don't really care, to be honest. Your call.

Reply:
My slight preference is that you accept the suggestion. =) It is more readable and I think if someone is sending negative floats to this function, they should get an error back.